packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -206,8 +207,8 @@ export function makeRumPublicApi(
 
     addAction: monitor((name: string, context?: object) => {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I am wondering if it could not be valuable to try to systematically sanitize all public API params with a sanitize wrapper.
It would avoid any miss and move most of the sanitize calls to public api files.

wdyt?

Reply:
Agreed - but some functions expect some parameters (notably errors) to be present/untouched. I would tackle this outside of this PR.

Reply:
Should we do something in this PR with the inputs that you are currently not serializing?

Reply:
Added sanitize to `name` of `addTiming`. We should tackle all inputs that can be sanitized with this PR.